app/src/org/commcare/dalvik/activities/CommCareSetupActivity.java

```diff
@@ -174,8 +174,6 @@ protected void onCreate(Bundle savedInstanceState) {
                     this.uiState = UiState.READY_TO_INSTALL;
                     //Now just start up normally.
                 }
-            } else {
-                incomingRef = this.getIntent().getStringExtra(KEY_PROFILE_REF);

```

Comment:
only place `KEY_PROFILE_REF` is ever used. So the result is always going to be `null`, so just remove it